### ISSUE TITLE
AV-8B Composite gauges broken at 1.6.5011 

### ISSUE THREAD

**Issue by BlueFinBima** [HeliosVirtualCockpit/Helios#486]
Reported by Torattacker on Discord as the AV-8BV4 profile not working, however simplest test case is to use the `Master Arm Panel` from the `AV-8B Gauges` toolbox bound to the AV8B interface.  On load the following error is issued.  For the AV-8BV4 profile, there are a huge number of this type of error.

```
2021-11-09 09:17:24.1650|ERROR|GadrocsWorkshop.Helios.HeliosSerializer|Binding source trigger "Master Arm Panel_Master Arm.Master Arm Panel_Master Arm.position.changed" not found at path "Visual;Monitor 1.Master Arm Panel.Master Arm Panel_Master Arm;Helios.Base.ToggleSwitch;Master Arm Panel_Master Arm"|
2021-11-09 09:17:24.1650|WARN|GadrocsWorkshop.Helios.HeliosSerializer|Dangling binding discarded on Profile load|
```
This binding was bound prior to 1.6.5011.

**Comment by BlueFinBima** [comment]
TriggerID for autobound HeliosValue triggers are no longer matching the dictionary key check due to duplicated prefixes on the TriggerID.  The TriggerID is being changed asynchronously, and appears to be triggered by the `action.Device = device;`   at line 355 in CompositeVisual.cs.  I guess it is part of the processing in HeliosObjectCollection / KeyedObservableCollection
Probably was working badly prior to a refactoring.
More work needed.

**Comment by BlueFinBima** [comment]
Notes:  Problem was introduced by the changes to UpdateId() in HeliosValue.cs and HeliosTrigger.cs in Commit 661c7e8b .

**Comment by BlueFinBima** [comment]
And fixed at 1.6.5101 https://github.com/HeliosVirtualCockpit/Helios/releases/tag/1.6.5101.0
